### ISSUE TITLE
Resnet50 training loss不下降

### ISSUE THREAD

**Issue by chaofanl** [PaddlePaddle/PaddleClas#2940]
目前基于paddleclas 在imagenet2012 上面 训练resnet50  发现loss 不下降
<img width="441" alt="image" src="https://github.com/PaddlePaddle/PaddleClas/assets/50936814/aa02107e-77bf-4c8a-b50e-673ce43c036c">

为了验证kernel，特地跑了下resnet18  ：
<img width="428" alt="image" src="https://github.com/PaddlePaddle/PaddleClas/assets/50936814/73dbc25b-fec5-4448-acb4-52bd25a75170">

resnet50和resnet18的配置一样的， 所以想问下大概是哪个出了问题，期待你们的回复，谢谢！
配置如下：
batch size 两个网络都是用的 256 ，都是单卡
Arch:
  name: ResNet50
  class_num: 1000
  input_image_channel: *image_channel
  data_format: "NHWC"
 
Loss:
  Train:
    - CELoss:
        weight: 1.0
  Eval:
    - CELoss:
        weight: 1.0
Optimizer:
  name: SGD
  lr:
    name: Piecewise
    learning_rate: 0.1
    decay_epochs: [30, 60, 90]
    values: [0.1, 0.01, 0.001, 0.0001]
  regularizer:
    name: 'L2'
    coeff: 0.0001

**Comment by cuicheng01** [comment]
可以提供下paddle和paddleclas版本吗？
